### PR TITLE
Improve vtrust

### PR DESCRIPTION
**Pull request by RusticLuftig**
V-trust improvements:
1. Improving the dataset loader to get more samples (No longer use "MultiChoiceRewardModel_rewards" which was causing a lot of samples to be dropped by wandb. I don't know why)
2. Aligning validator evaluations to "sync blocks" to reduce the variance in evaluations performed across validators
3. Switch to a WTA weight rather than softmax
4. Set weight to 0 for models with very little weight

Other improvements:
1. Correctly handle set_weight errors
2. Since we're now rate limiting eval loops, increases the number of models per loop to 20.

### CODE REVIEW

File: finetune/datasets/subnet/prompting_subset_loader.py [hunk 1]
```diff
@@ -74,178 +79,177 @@ def _get_filters(
...
         prompting_project: str = constants.PROMPTING_WANDB_PROJECT,
-        max_run_age: typing.Optional[dt.timedelta] = None,
-        min_percent_correct: typing.Optional[float] = None,
+        oldest_sample_timestamp: typing.Optional[dt.datetime] = None,
+        newest_sample_timestamp: typing.Optional[dt.datetime] = None,
         validator_hotkeys: typing.Optional[typing.Set[str]] = None,
     ):
         """Loads prompt/response data from Subnet 1.
 
+        Please note: this loader assumes that it's goign to fetch recent data (past few hours) and will likely perform poorly if you try to fetch data from a long time ago.
```
Comment (Sid-Data-Universe) [inline finetune/datasets/subnet/prompting_subset_loader.py:91]: going

File: neurons/config.py [hunk 2]
```diff
@@ -40,16 +40,10 @@ def validator_config():
...
-        "--latest_prompting_steps",
-        type=int,
-        default=500,  # Sample more steps since prompting runs this less frequently.
-        help="Number of most recent Prompting steps to sample data from",
-    )
     parser.add_argument(
         "--latest_prompting_samples",
         type=int,
-        default=400,
+        default=500,
```
Comment (Sid-Data-Universe) [inline neurons/config.py:46]: Go further up? 600/800?

**Review by Sid-Data-Universe — APPROVED** [review]

**Review by Sid-Data-Universe — APPROVED** [review]
